measures/2021/measures-data.json

```diff
@@ -10565,6 +10565,38 @@
     "measureSets": [],
     "measureSpecification": {}
   },
+  {
+    "title": "All-Cause Unplanned Admissions for Patients with Multiple Chronic Conditions for ACOs (MCC)",
+    "eMeasureId": null,
+    "nqfEMeasureId": null,
+    "nqfId": null,
+    "measureId": "MCC1",
+    "description": "Rate of risk-standardized acute, unplanned hospital admissions among Medicare fee-for-service (FFS) beneficiaries 65 years and older with multiple chronic conditions (MCCs) who are assigned to the Accountable Care Organization (ACO)",
+    "nationalQualityStrategyDomain": "Outcome",
+    "measureType": "outcome",
+    "primarySteward": "Centers for Medicare & Medicaid Services (CMS)",
+    "metricType": "singlePerformanceRate",
+    "firstPerformanceYear": 2021,
+    "lastPerformanceYear": null,
+    "isHighPriority": true,
+    "isInverse": true,
+    "isIcdImpacted": false,
+    "isClinicalGuidelineChanged": false,
+    "category": "quality",
+    "isRegistryMeasure": false,
+    "isRiskAdjusted": false,
+    "icdImpacted": [],
+    "clinicalGuidelineChanged": [],
+    "allowedPrograms": [
+      "mips",
```

Comment:
FYI, I think this is only for APP1 in Y5...

Shad  11:55 AM
Hey quick question for MCC1... Is it also going to be required for Y5 MIPS WI submissions similar to Y5 APP WI submissions?

steven_szeliga  12:01 PM
nope, its going into the repo but its only for APP and will only be delivered for ACOs, its being delivered by a 3rd party

Reply:
Does it matter if "Outcome" is capitalized for "nationalQualityStrategyDomain": "Outcome"
and not capitalized for "measureType": "outcome"?